Doc/library/stdtypes.rst

```diff
@@ -1687,8 +1687,8 @@ expression support in the :mod:`re` module).
    Return true if the string is a valid identifier according to the language
    definition, section :ref:`identifiers`.
 
-   Use :func:`keyword.iskeyword` to test for reserved identifiers such as
-   :keyword:`def` and :keyword:`class`.
+   Call :func:`keyword.iskeyword` to test whether string `s` is a reserved
```

Comment:
To test whether string `s` is a reserved identifier you have to call `keyword.iskeyword(s)`, not `keyword.iskeyword()`.

I don't think anything should be changed here. Currently the link removes even the smallest chance of misunderstanding it. IMHO changes will make the documentation worse.

Reply:
In my suggestion on the issue, I specifically included the argument '(s)' in the call so that 's' later in the sentence referred back to the string passed as an argument.  I changes the vague 'use' to the specific 'call'.  'test for reserved identifiers' is awkward.  Test what? (especially with the subject s never mentioned).  Changes are not always bad.

I like the following, based on Serhiy's comment, even better as a followup to the preceding sentence.

`To test whether string s is a reserved keyword, such as def or class, call keyword.iskeyword(s).`

Reply:
Sure, I will change to that. The only issue I faced adding an argument in the function is that it is not hyperlinked in the html docs if I did :func:`keyword.iskeyword(s)`. Although :func:`keyword.iskeyword` works fine.

I searched the docs for a similar function reference that accepts an argument but I couldn't find. Can you please guide me on how do I change it so that it is also hyperlinked to the actual method definition in the html docs?

Reply:
This is the reason why I suggest to not change the current documentation. It is good enough and changes make it worse: technically incorrect, more verbose, or less informative.

Reply:
CuriousLearner: I presume, but don't know for sure, that you can pur '(x)' outside the backticks and get the hyperlink.

Serhiy: Either proposed change, the second of which is based on your statement, is more informative, not less, and more technically correct, not less.  Your said it yourself.  iskeyword "test[s] whether string s *is* a reserved identifier [singular]", whereas the somewhat cryptic 'test for reserved identifiers [plural]' can easily be expanded to and construed as meaning 'test whether string s *contains* reserved identifiers [plural]".  The IDLE colorizer does just that.  It 'tests for reserved keywords', in this sense,  and when it finds them, tags them.

Reply:
@terryjreedy I already tried that & it does not get hyperlinked correctly.

@Mariatta Can you help here please?